### ISSUE TITLE
Error generating change log using >=1.16.0

### ISSUE THREAD

**Issue by jackton1** [github-changelog-generator/github-changelog-generator#956]
**Describe the bug**
Experiencing issues after upgrading from `1.15.2` -> `1.16.x`

```bash
Output: HISTORY.md
196 /usr/local/bundle/gems/octokit-4.20.0/lib/octokit/client/issues.rb:30:in `list_issues': wrong number of arguments (given 3, expected 0..2) (ArgumentError)
197 from /usr/local/bundle/gems/github_changelog_generator-1.16.1/lib/github_changelog_generator/octo_fetcher.rb:415:in `block in iterate_pages'
198 from /usr/local/bundle/gems/retriable-3.1.2/lib/retriable.rb:61:in `block in retriable'
199 from /usr/local/bundle/gems/retriable-3.1.2/lib/retriable.rb:56:in `times'
200 from /usr/local/bundle/gems/retriable-3.1.2/lib/retriable.rb:56:in `retriable'
201 from /usr/local/bundle/gems/github_changelog_generator-1.16.1/lib/github_changelog_generator/octo_fetcher.rb:446:in `check_github_response'
202 from /usr/local/bundle/gems/github_changelog_generator-1.16.1/lib/github_changelog_generator/octo_fetcher.rb:415:in `iterate_pages'
203 from /usr/local/bundle/gems/github_changelog_generator-1.16.1/lib/github_changelog_generator/octo_fetcher.rb:164:in `fetch_closed_issues_and_pr'
204 from /usr/local/bundle/gems/github_changelog_generator-1.16.1/lib/github_changelog_generator/generator/generator.rb:144:in `fetch_issues_and_pr'
205 from /usr/local/bundle/gems/github_changelog_generator-1.16.1/lib/github_changelog_generator/generator/generator.rb:55:in `block in compound_changelog'
206 from /usr/local/bundle/gems/async-1.29.0/lib/async/task.rb:263:in `block in make_fiber'
207 Using these options:
208 :date_format=>"%Y-%m-%d"
209 :output=>"HISTORY.md"
```

**To Reproduce**
Steps to reproduce the behaviour:

```bash
github_changelog_generator -u my-org -p my-repo -t "my-token" --base "" --output="HISTORY.md"
```

**Expected behavior**
Running the command above doesn't result in any errors

**Comment by jackton1** [comment]
https://github.com/github-changelog-generator/github-changelog-generator/issues/952

**Comment by olleolleolle** [comment]
Thanks, this issue has been fixed by #949, which will be in an upcoming release. You can use that unreleased code today, using the gem specific_install or Bundler git gem support, or by waiting for a new release.
